ci/docker/runtime_functions.sh

```diff
@@ -1781,7 +1782,7 @@ build_julia_docs() {
    export LD_PRELOAD='/usr/lib/x86_64-linux-gnu/libjemalloc.so'
    export LD_LIBRARY_PATH=/work/mxnet/lib:$LD_LIBRARY_PATH
 
-   julia_doc_path='julia/docs/site/'
+   julia_doc_path='julia/docs/site/'f
```

Comment:
is this intended?

Reply:
No, it was an mistake(thus casuing the check failed). I didn't realize that when commiting.
I will remove it now.